man/systemd.exec.xml

```diff
@@ -1682,10 +1682,31 @@
       </varlistentry>
 
       <varlistentry>
-        <term><varname>RuntimeDirectoryMode=</varname></term>
+        <term><varname>StateDirectory=</varname></term>
+        <term><varname>CacheDirectory=</varname></term>
+        <term><varname>LogsDirectory=</varname></term>
+        <term><varname>ConfigurationDirectory=</varname></term>
+
+        <listitem><para>Takes a whitespace-separated list of directory names. If set, as similar to
+        <varname>RuntimeDirectory=</varname>, one or more directories including their parents by the specified names
+        will be created below <filename>/var/lib</filename>, <filename>/var/cache</filename>, or <filename>/var/log</filename>,
+        respectively, when the unit is started. These options are ignored for user services.
```

Comment:
hmm, I think we should support this for user services too. it's easy and I think ultimately useful there too